system/infra-monitoring/vendor/atlas/templates/configmap.yaml

```diff
@@ -270,7 +270,7 @@ data:
               target: 2
               region: "{{ .Values.global.region }}"
               status: "active"
-              q: "cp"
+              q: "-cp"
```

Comment:
this one is not correct. `cp/netbox` is only used by ipmi-exporter and my guess would be, that you need this change for the bios-exporter?

Reply:
yes, that's tight, I changed the cp/netbox as well as it is anyways wrong, and it will have the same issue pulling servers that are not CP, but if you want me to undo that I will do it.

Reply:
Good point tho and good catch ;)